src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.service;
+
+import mate.academy.model.User;
+
+public interface UserService {
+    User add(User user);
+
+    User get(Long id);
+
+    User get(String email);
+}
```

Comment:
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/49767049/fd9cd4dd-d484-4691-baaf-10f84dd0a52f)
here, service should return optional
it's expected situation - not to be able to find a user by email, we shouldn't throw exception